### PR TITLE
Loss function arguments for varimp

### PR DESCRIPTION
**Pull request by rachaelvp**
Loss function arguments were provided in the wrong order in `varimp`. 

I am not sure why this PR contains commit from another PR. Any fix for how to avoid this in the future?

### CODE REVIEW

**Comment by rachaelvp** [comment]
Hey @nhejazi, could you help me identify what I need to do in order for this build to pass? Also, do you happen to know why prior PRs from different branches were incorporated in this PR, and how to fix this?

**Comment by nhejazi** [comment]
The log indicates that the failed builds are actually due to a change in Keras and/or python3 that causes `Lrnr_lstm` to fail (https://travis-ci.com/tlverse/sl3/jobs/289536647#L3209). This itself is a result of a change I introduced in https://github.com/tlverse/sl3/commit/4f5a63c95bfe99cdd71b77a3f04a6d08774d70a6, which itself fixes a build error due to python2 being at it's end of life.

As for the commits from other PRs, this is likely due to creating new branches starting at the commits included in those PRs. That is, to avoid this going forward, a good way to operate is to switch back to the `devel` branch prior to starting a new PR.

**Comment by nhejazi** [comment]
Should be merged since Appveyor passed
